programs/validator-history/src/instructions/copy_gossip_contact_info.rs

```diff
@@ -2,12 +2,34 @@ use anchor_lang::{
     prelude::*,
     solana_program::{self, clock::Clock, pubkey::Pubkey, sysvar},
 };
+use bytemuck::{from_bytes, Pod, Zeroable};
 
 use crate::{
-    crds_value::CrdsData, errors::ValidatorHistoryError, state::ValidatorHistory, utils::cast_epoch,
+    crds_value::CrdsData, errors::ValidatorHistoryError, state::ValidatorHistory,
+    utils::cast_epoch, Config,
 };
 use validator_history_vote_state::VoteStateVersions;
 
+// Structs and constants copied from solana_sdk::ed25519_instruction. Copied in order to make fields public. Compilation issues hit when importing solana_sdk
+#[derive(Default, Debug, Copy, Clone, Zeroable, Pod, Eq, PartialEq)]
+#[repr(C)]
+pub struct Ed25519SignatureOffsets {
```

Comment:
why can't import thes from solana_sdk?

Reply:
nvmd

Reply:
Running into some bad version conflicts on inner packages of solana-sdk when mixing it with anchor. have hit this before with things like the gossip structs and we ended up copying those over as well